.circleci/build-docs.sh

```diff
@@ -58,7 +58,9 @@ rm -r *
 
 # build docs and copy over to tmpdir
 cd ${DOCSOURCE}
-make clean html SPHINXOPTS="-j2" 2>&1 | grep -v "WARNING: nonlocal image URL found:"
+# TODO: reenable "-j2" when docs build fine
```

Comment:
Looking at the build times, it doesn't make much sense to use multiple jobs here. It's less than a minute faster with `-j2` but also "obfuscates" the progress/error output..